site/package.json

```diff
@@ -57,7 +57,7 @@
 		"@xterm/addon-webgl": "0.18.0",
 		"@xterm/xterm": "5.5.0",
 		"ansi-to-html": "0.7.2",
-		"axios": "1.7.4",
+		"axios": "1.7.2",
```

Comment:
1.7.2 is vulnerable.

Reply:
it was just a merge conflict, I got them all cleaned up